doc/system_description/boards.md

```diff
@@ -1,5 +1,8 @@
 # Tillitis PCB designs
 
+**NOTE:** Documentation migrated to dev.tillitis.se, this is kept for
```

Comment:
The boards document isn't in Dev Handbook yet. Maybe it should stay here and not have any warning?

Reply:
Good catch. Will remove the note.